Backend/dev/test-tool/context-api/server.py

```
@@ -4348,6 +4359,20 @@ def sh_capture(cmd, timeout=8):
+        elif path == "/api/load-test/tokens":
+            tokens_dir = Path(__file__).parent.parent.parent.parent / "load-test" / "tokens"
+            try:
+                riders  = json.loads((tokens_dir / "riderTokens.json").read_text())
+                drivers = json.loads((tokens_dir / "driverTokens.json").read_text())
+                self._send_json({
+                    "riders": riders,
+                    "drivers": drivers,
+                    "riderCount": len(riders),
+                    "driverCount": len(drivers),
+                })
+            except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)